app/models/osbb.rb

```diff
@@ -0,0 +1,14 @@
+class Osbb < ApplicationRecord
+  before_save :downcase_email
+  
+  validates :name, presence: true, length: { minimum: 3, maximum: 255 }
+  validates :phone, :postcode,  numericality: { only_integer: true }
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+  validates :email, length: { maximum: 255 },
+  format: { with: VALID_EMAIL_REGEX }
```

Comment:
please write this validation in one line, like one above

Reply:
Resolve